ui/pages/confirmation/confirmation.js

```diff
@@ -147,7 +147,9 @@ export default function ConfirmationPage({
   };
 
   ///: BEGIN:ONLY_INCLUDE_IN(flask)
-  const snaps = useSelector(getSnaps);
+  const {
+    manifest: { snapName: proposedName },
+  } = useSelector(getSnap);
```

Comment:
You need to pass the snapId to the selector

Reply:
Oh yeah 😅

Reply:
https://github.com/MetaMask/metamask-extension/blob/4082c33eb62a2411512e54c619a4504e70283ac0/ui/components/app/flask/snap-ui-renderer/snap-ui-renderer.js#L62